tests/unit/dataactvalidator/test_sf133_content.py

```diff
@@ -0,0 +1,161 @@
+                    '1018', '1019', '1020', '1021', '1022', '1023', '1024', '1025', '1026', 
+                    '1027', '1028', '1029', '1030', '1031', '1032', '1033', '1034', '1035', 
+                    '1036', '1037', '1038', '1039', '1040', '1041', '1042', '1160', '1180', 
+                    '1260', '1280', '1340', '1440', '1540', '1640', '1750', '1850'
+                ]
+
+                sum_two = 0
+                validate_two = 0
+                validate_two_lines = ['2190', '2490']
+                for index, row in current_tas.iterrows():
+                    if row.line in validate_one_lines:
+                        sum_one += row.amount
+                    elif row.line=='1910':
+                        validate_one = float("{0:.2f}".format(row.amount))
+                    elif row.line in validate_two_lines:
+                        sum_two += row.amount
+                    elif row.line=='2500':
+                        validate_two = float("{0:.2f}".format(row.amount))
+                sum_one = float("{0:.2f}".format(sum_one))
+                sum_two = float("{0:.2f}".format(sum_two))
+
+                validation_one = sum_one==validate_one
+                validation_two = sum_two==validate_two
+                validation_three = validate_one==validate_two
+
+                tas_m = TAS_RE.match(tas)
+
+                # initial validations fail
+                if not (validation_one and validation_two) and (tas_m and tas_m.group('atc')!='X' and file_fiscal_year>int(tas_m.group('epoa'))):
+                    if not validation_three:
+                        failed_validations.append('{0},{1},{2},{3},{4},{5},{6},{7},1900!=2500,{8},{9}'.format(
+                            file_index, tas_m.group('ata'), tas_m.group('aid'), 
+                            tas_m.group('atc'), tas_m.group('bpoa'), tas_m.group('epoa'), 
+                            tas_m.group('mac'), tas_m.group('sac'), validate_one, validate_two)
+                        )
+                else:
+                    if not validation_one:
+                        failed_validations.append('{0},{1},{2},{3},{4},{5},{6},{7},sum!=1910,{8},{9}'.format(
+                            file_index, tas_m.group('ata'), tas_m.group('aid'), 
+                            tas_m.group('atc'), tas_m.group('bpoa'), tas_m.group('epoa'), 
+                            tas_m.group('mac'), tas_m.group('sac'), sum_one, validate_one)
+                        )
+                    if not validation_two:
+                        failed_validations.append('{0},{1},{2},{3},{4},{5},{6},{7},sum!=2500,{8},{9}'.format(
+                            file_index, tas_m.group('ata'), tas_m.group('aid'), 
+                            tas_m.group('atc'), tas_m.group('bpoa'), tas_m.group('epoa'), 
+                            tas_m.group('mac'), tas_m.group('sac'), sum_two, validate_two)
+                        )
```

Comment:
As we add more validations, lots of `if-append`s will be hard to understand. What do you think about moving these validations to separate functions and then running each in sequence?